requirements.txt

```diff
@@ -1,3 +1,4 @@
+# TODO: Update with exact module version
 numpy
 torch
 opencv_python
```

Comment:
Is it possible to not make `opencv_python` a a requirement here?

Sometimes users will use other methods to install OpenCV, just as detectron2 did [that](https://github.com/facebookresearch/detectron2/blob/40f8cffa7eb997a8a1fff71bfae728fabce8b20a/setup.py#L166)

> Just like PyTorch, user should install opencv themselves, preferrably by OS's package manager, or by choosing the proper pypi package name at https://github.com/skvark/opencv-python.

Then add an opencv installation instruction in [Readme](https://github.com/Megvii-BaseDetection/YOLOX/blob/main/README.md#quick-start).

Reply:
Yes, It can be done. I have removed `opencv-python` from `requirements.txt` and added Instructions in Readme to install it manually.

Reply:
@FateScript is the maintainer, His thoughts are more important, let's wait for the conclusion about this

Reply:
I believe that opencv_python is neccessry in requirments.txt, thx for noting in README  , but I don't think it's a proper way @imneonizer

Reply:
I have added it back to requirments.txt aswell

Reply:
> I have added it back to requirments.txt aswell

plz also revert README, thx

Reply:
Reverted